### PR TITLE
README.md Add info how to check the device number

### LINKED ISSUE #462: README doesn't specify how to check which video device is added
> https://stackoverflow.com/questions/tagged/v4l2loopback

As a completely new user, I could ask there, but after like 30 minutes reading various docs I still don't have an answer how to check if `modprobe v4l2loopback` command actually added any devices, and what are their numbers. There are three devices.

```
➜  ~ ls -a  /dev | grep video
video0
video1
video2
```
Running `modprobe v4l2loopback` again doesn't add anything new.

It is not a problem to send a PR to add the missing info what is the right way to see that `modprobe v4l2loopback` command added devices successfully, how to check which devices were added, and how to check they actually work. The missing thing is the knowledge of the right way.

### Step 2: Describe your environment

  * `v4l2loopback` version: _____

```
[    5.603674] v4l2loopback: loading out-of-tree module taints kernel.
[    5.603723] v4l2loopback: module verification failed: signature and/or required key missing - tainting kernel
[    5.604675] v4l2loopback driver version 0.12.5 loaded
```

  * kernel version: 5.16.8-200.fc35.x86_64
  * Distribution (+version): Fedora 35

### PR DESCRIPTION
**Pull request by abitrolly**
Closes #462.

Although this shows how to figure out the device number, it still doesn't reveal how to test that the loopback device works.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,20 +110,25 @@ Using `sudo` use:
...
-fed by various programs.
+This will create an additional video-device, e.g. `/dev/video0` (the number
+depends on whether you already had video devices on your system). You can
+check which device is created by listing this directory contents.
+
+    $ ls -1 /sys/devices/virtual/video4linux
+    video0
+
+Various programs can then feed content to the loopback device `/dev/video0`.
+
```
Comment (umlaeute) [inline README.md:121]: i think this is confusing: the loopback device can be `/dev/video0` or something else (i guess you refer to the `video0` in the previous line, but this is not clear.

File: README.md [hunk 2]
```diff
@@ -110,20 +110,25 @@ Using `sudo` use:
...
-In theory most programs capable of _writing to_ a v4l2 device should work.
+In theory most programs capable of _writing to_ a `v4l2` device should work.
 
-The data sent to the v4l2loopback device can then be read by any v4l2-capable
+The data sent to the `v4l2loopback` device can then be read by any v4l2-capable
 application.
 
-You can find a number of scenarios on the wiki at
-	http://github.com/umlaeute/v4l2loopback/wiki
+See wiki at https://github.com/umlaeute/v4l2loopback/wiki for possible scenarios.
```
Comment (umlaeute) [inline README.md:131]: the changes in lines 123..131 seem to be unrelated. please remove them.

File: README.md (hunk 1 above)
Comment (abitrolly) [reply README.md:122]: Is that better now?

Comment (umlaeute) [reply README.md:122]: i still dont like the wording of the last sentence, as it fixates `/dev/video0`
i think the original wording was better in this respect.

How about simply adding the "You can check..." (until the example) after the current wording? (and add a note that the devices you see can be used under `/dev`).
e.g.

~~~
You can see which loopback devices are created by checking the `/sys/devices/virtual/video4linux` directory.
E.g. if there were two v4l2loopback devices `/dev/video0` and `/dev/video3` you would get:

    $ ls -1 /sys/devices/virtual/video4linux
    video0
    video3
~~~

**Review by umlaeute — CHANGES_REQUESTED** [review]
please also do squash your commits and force-push them

(instead of an overzealous commit + an extra commit that undoes the zeal, i'd rather have a single clean commit; the rest can be squashed as well...)

**Comment by abitrolly** [comment]
Reworded and squashed.
